README.md

```diff
@@ -18,6 +18,26 @@ composer require tightenco/duster --dev
 ./vendor/bin/duster init
 ```
 
+When installing you may see the following message:
+
+>dealerdirect/phpcodesniffer-composer-installer contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
+>Do you trust "dealerdirect/phpcodesniffer-composer-installer" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?]
+
+You will need to accept the [phpcodesniffer-composer-installer](https://github.com/PHPCSStandards/composer-installer) to have the PHPCS rulesets and GitHub actions to work.
```

Comment:
I think this grammar isn't quite what you intended. Do you mean "to have the PHPCS rulesets and so the GitHub Actions will work"?

Reply:
Yes, thanks!  I've updated the readme.